src/css/profile/mobile/common/core.less

```diff
@@ -165,6 +170,12 @@
 	}
 }
 
+body.ui-theme-dark {
```

Comment:
Is there no need to consider light theme?
The followings code is for light theme?

	.ui-content-area {
		background-color: var(--background-area-color);
		mask-border: url(images/0_Round_corner/round.svg) 77;
		mask-border-width: 26 * @px_base;
		border-radius: 26 * @px_base;
		box-sizing: border-box;
		box-shadow: 0 0 0 0.25 * @px_base var(--content-area-line-color) inset;
	}

Reply:
In 51 page, mask does not need to be applied to dark theme.

Reply:
ok. I see~